Content.Server/Administration/BwoinkSystem.cs

```diff
@@ -60,15 +97,56 @@ protected override void OnBwoinkTextMessage(BwoinkTextMessage message, EntitySes
             foreach (var channel in targets)
                 RaiseNetworkEvent(msg, channel);
 
+            var sendsWebhook = _webhookUrl != string.Empty;
+            if (sendsWebhook)
+            {
+                var sawmill = IoCManager.Resolve<ILogManager>().GetSawmill("AHELP");
+
+                void LookupFinished(Task<LocatedPlayerData?> finishedTask)
+                {
+                    if (finishedTask.Result == null)
+                    {
+                        sawmill.Log(LogLevel.Error, $"Unable to find player for netuserid {msg.ChannelId} when sending discord webhook.");
+                        return;
+                    }
+
+                    var payload = new WebhookPayload()
+                    {
+                        Username = _serverName,
+                        Content = $"`[{finishedTask.Result.Username}]` {senderSession.Name}: \"{escapedText}\""
+                    };
+                    var request = _httpClient.PostAsync(_webhookUrl,
+                        new StringContent(JsonConvert.SerializeObject(payload), Encoding.UTF8, "application/json"));
```

Comment:
Please use `System.Text.Json` here instead of `Newtonsoft.Json`. Then you can also use `PostAsJson()` instead.

Reply:
only reason i used it here is bc i took the ooc relay as example code. will fix